task-list/config/routes.rb

```diff
@@ -1,16 +1,21 @@
 Rails.application.routes.draw do
 
-  #root 'tasks#index'
-
-  get 'tasks/' => 'tasks#index'
+  root 'tasks#index'
```

Comment:
This seems like a good choice overall! Now many of your controllers can simply `redirect_to '/'` and we don't have to worry about whether or not your routes use `tasks/` or `todo/` or `task_list/` or something else.